### PR TITLE
[cloud-provider-yandex][docs] Setting custom storageclass as default FAQ

### PR DESCRIPTION
**Pull request by meatuses**
## Description
Add FAQ paragraph on how to set custom StorageClass as default
## Why do we need it, and what problem does it solve?
Original issue https://github.com/deckhouse/deckhouse/issues/6213

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-provider-yandex
type: chore
summary: Minor documentation updates.
impact_level: low
```

### CODE REVIEW

**Comment by z9r5** [comment]
@kirkonru Hi! Do a review, please.

File: modules/030-cloud-provider-yandex/docs/FAQ_RU.md [hunk 1]
```diff
@@ -44,3 +44,19 @@ reserved: true
...
+
+Для назначения произвольного StorageClass'а используемым по умолчанию, выполните следующие шаги:
+
+1. Добавьте на StorageClass аннотацию `storageclass.kubernetes.io/is-default-class='true'`:
+
+   ```shell
+   kubectl annotate sc $STORAGECLASS storageclass.kubernetes.io/is-default-class='true'
+   ```
+
+2. Укажите имя StorageClass'а в параметре [storageClass.default](configuration.html#parameters-storageclass-default) в настройках модуля `cloud-provider-yandex`. Обратите внимание, что после этого аннотация `storageclass.kubernetes.io/is-default-class='true'` снимется со StorageClass, который был указан в настройках модуля ранее как используемый по умолчанию.
```
Comment (kirkonru) [inline modules/030-cloud-provider-yandex/docs/FAQ_RU.md:58]:
```suggestion
2. Укажите имя StorageClass'а в параметре [storageClass.default](configuration.html#parameters-storageclass-default) в настройках модуля `cloud-provider-yandex`. Обратите внимание, что после этого аннотация `storageclass.kubernetes.io/is-default-class='true'` снимется со StorageClass, который ранее был указан в настройках модуля как используемый по умолчанию.
```

File: modules/030-cloud-provider-yandex/docs/FAQ.md [hunk 2]
```diff
@@ -44,3 +44,19 @@ Pay attention to the following nuances:
 If the dhcpOptions parameter is set, all DNS are routed to the DNS servers specified. These DNS servers **must** serve DNS requests to the Internet and (if needed) resolve intranet resources.
 
 **Do not use** this option if the recursive DNSs specified cannot resolve the same list of zones that the recursive DNSs in the Yandex Cloud subnet can resolve.
+
+## How to set custom StorageClass as default?
+
+Do the following steps to set a custom StorageClass as default:
```
Comment (kirkonru) [inline modules/030-cloud-provider-yandex/docs/FAQ.md:50]:
```suggestion
Do the following to set a custom StorageClass as default:
```

File: modules/030-cloud-provider-yandex/docs/FAQ.md [hunk 3]
```diff
@@ -44,3 +44,19 @@ Pay attention to the following nuances:
 If the dhcpOptions parameter is set, all DNS are routed to the DNS servers specified. These DNS servers **must** serve DNS requests to the Internet and (if needed) resolve intranet resources.
 
 **Do not use** this option if the recursive DNSs specified cannot resolve the same list of zones that the recursive DNSs in the Yandex Cloud subnet can resolve.
+
+## How to set custom StorageClass as default?
```
Comment (kirkonru) [inline modules/030-cloud-provider-yandex/docs/FAQ.md:48]:
```suggestion
## How to set a custom StorageClass as default?
```

File: modules/030-cloud-provider-yandex/docs/FAQ.md [hunk 4]
```diff
@@ -44,3 +44,19 @@ Pay attention to the following nuances:
...
+
+Do the following steps to set a custom StorageClass as default:
+
+1. Add `storageclass.kubernetes.io/is-default-class='true'` annotation to the StorageClass:
+
+   ```shell
+   kubectl annotate sc $STORAGECLASS storageclass.kubernetes.io/is-default-class='true'
+   ```
+
+2. Specify the StorageClass name in the [storageClass.default](configuration.html#parameters-storageclass-default) parameter in the `cloud-provider-yandex` module settings. Note that after that, the `storageclass.kubernetes.io/is-default-class='true'` annotation will be removed from the StorageClass, which was previously specified in the module settings as the default.
```
Comment (kirkonru) [inline modules/030-cloud-provider-yandex/docs/FAQ.md:58]:
```suggestion
2. Specify the StorageClass name in the [storageClass.default](configuration.html#parameters-storageclass-default) parameter in the `cloud-provider-yandex` module settings. Note that after doing so, the `storageclass.kubernetes.io/is-default-class='true'` annotation will be removed from the StorageClass that was previously listed in the module settings as the default one.
```

**Review by kirkonru — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6280](https://github.com/deckhouse/deckhouse/pull/6280) to the branch [release-1.53](https://github.com/deckhouse/deckhouse/tree/release-1.53) successful!

**Pull request merged**